apkid/rules/apk/packers.yara

```diff
@@ -604,3 +604,23 @@ rule secenh : packer
     and 1 of ($a*)
     and 1 of ($b*)
 }
+
+// There can also be rules for dex files because of NRV compression but I can't find compressed classesN.dex
+// I guess it is optional(?).
+rule legu : packer
+{
+  meta:
+    description = "Legu Packer"
+    sample = "9ff3a53f76c7a6d7e3de3b8567c9606f2cc08ec4aaaae596a27361018d839c58"
+    author = "Mert Arıkan"
+  strings:
+    $a = "assets/toversion"
+    $b = "assets/0OO00l111l1l"
+    $c = "assets/0OO00oo01l1l"
+    $d = "assets/o0oooOO0ooOo.dat"
+  condition:
+    is_apk
+    and $b 
+    and ($a or $c or $d)
+
```

Comment:
Can you remove the extra-whitespace?

Reply:
Can you also adjust properly the spaces?

![Screenshot from 2019-12-23 10-35-55](https://user-images.githubusercontent.com/14809754/71349770-11000400-2570-11ea-9d31-88b0a7af208e.png)

Reply:
First commit was actually edited via Github's own Edit functionality. So, it messed up with first commit :) It was fixed on second commit